### PR TITLE
RethinkDB 2.1.2 and 2.0.5 release notes

### PR DESCRIPTION
**Pull request by danielmewes**
We're going to do two point releases, one on the 2.0 branch and one for 2.1.

@mglukhovsky could you review please?

### CODE REVIEW

File: NOTES.md [hunk 1]
```diff
@@ -1,3 +1,60 @@
+# Release 2.1.2 (Forbidden Planet)
+
+Released on 2015-08-24
+
+Bug fix release
+
+### Compatiblity ###
+
+* RethinkDB 2.1.2 servers cannot be mixed with servers running earlier versions in the
+  same cluster.
```
Comment (danielmewes) [inline NOTES.md]: Inconsistent punctuation.

File: NOTES.md [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+* Changefeeds on a `get_all` query no longer return initial values. This restores the
+  behavior from RethinkDB 2.0
+
+### Bug fixes ###
+
+* Fixed an issue that could cause writes to be acknowledged before all necessary data got
```
Comment (mglukhovsky) [inline NOTES.md]: Poor grammar. I suggest:

> Fixed an issue where writes could be acknowledged before all necessary data was written to disk.

**Comment by mglukhovsky** [comment]
One comment, otherwise it looks good!

**Comment by danielmewes** [comment]
Thanks, I updated the release notes.

I also added one more item:
"Improved how query execution times are reported in the Data Explorer (#4752)"

**Comment by danielmewes** [comment]
I merged this into `v2.1.x` so I can tag the release and start building.
@mglukhovsky please let me know if there was anything wrong with my last two additions:
```
* Out-of-memory errors in the server's JSON parser are now correctly reported (#4751)
* Improved how query execution times are reported in the Data Explorer (#4752)
```
... so we can improve them for the releases page.

**Comment by mglukhovsky** [comment]
Looks good to me!

**Pull request merged**